tests/bundle-test.ts

```diff
@@ -19,10 +21,11 @@ import type {
   SigningAlgorithms,
   SigningData,
 } from '@kiltprotocol/types'
-import type { KeypairType } from '@polkadot/util-crypto/types'
 
 const { kilt } = window
 
+const WS_PORT = 9944
```

Comment:
If we’re not reusing the value by exporting and importing, does it even make sense to introduce this const?

Reply:
I guess not